### DISCUSSION TITLE
win10+texlive2021+texstudio，参考文献无法显示

### DISCUSSION THREAD

**Discussion by VillardX** [sjtug/SJTUThesis#912] [提问]
将样例模板下载到本地，并按照指导进行Commands-Latexmk和Build-Default Compiler的修改。编译样例main.tex，可以生成对应的pdf文件，但pdf文件中没有显示参考文献。编译消息如下

'''
开始 : latexmk -silent -synctex=1 -xelatex "main"

Latexmk: Run number 1 of rule 'xelatex'
Latexmk: Run number 2 of rule 'xelatex'
Latexmk: Run number 1 of rule 'xdvipdfmx'
Latexmk: Summary of warnings from last run of *latex: Latex failed to resolve 27 citation(s)
完成

开始 : "D:/ZZ/SumatraPDF/SumatraPDF.exe" "main".pdf

出现错误
'''

环境为win10+texlive2021+texstudio

求助

**Comment by LogCreative** [comment]
请上传你的日志文件 main.log。

最好安装最新的 TeX 发行版（TeX Live 2023）后重试。

**Reply by VillardX** [reply]
[main.log](https://github.com/sjtug/SJTUThesis/files/11147219/main.log)

**Reply by LogCreative** [reply]
可以再提供 main.blg

**Reply by VillardX** [reply]
[main.zip](https://github.com/sjtug/SJTUThesis/files/11147743/main.zip)
无法直接上传blg文件，我进行了压缩

**Reply by LogCreative** [reply]
似乎仍然在使用 bibtex 编译，应当改为 biber；可以清理一下临时文件再重新使用 latexmk (xelatex) 试试。

**Reply by VillardX** [reply]
非常感谢！修改biber编译后能够正常显示参考文献。:)

**Reply by VillardX** [reply]
我发现了新的问题，目录中显示是直接从第一章开始的，请问如何在目录中加入摘要和Abstract呢？谢谢

**Reply by VillardX** [reply]
以及“参考文献”这四个字我看学校的模板是每个字空开来的，请问在哪里设置“参 \quad 考 \quad 文 \quad 献”呢；英文摘要页的abstract标题字体在哪里调整呢？我找了setup.tex和main.tex以及abstract.tex好像都没相关参数可以设置，谢谢~

**Reply by LogCreative** [reply]
#913
